src/components/Header/Header.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import './Header.css';
+import { Link } from 'react-router-dom';
+
+const Header = () => {
+  return (
+    <header className='header'>
+      <h1 className='veg-share-title'>Veg-Share</h1>
+      <div className='header-links'>
+        <Link className='profile-link' to='/profile'>Profile</Link>
```

Comment:
We could toggle between Profile and Home based on the view. Right now there isn't a way to get back to the home view when a user clicks on Profile.

Reply:
Good call! I was thinking originally that the Profile and Home links would appear conditionally based on the current page, but I skipped ahead by about 14 steps.